third_party/utf8proc/utf8proc_wrapper.cpp

```diff
@@ -30,71 +30,71 @@ static void AssignInvalidUTF8Reason(UnicodeInvalidReason *invalid_reason, size_t
 	}
 }
 
+#define EXTRA_BYTE_LOOP(NEXTRA_BYTES) \
```

Comment:
Can we avoid defines and use templates instead? :)

Reply:
I was predicting these :)